web/src/lib/store/metalakes/index.js

```diff
@@ -38,6 +38,8 @@ export const fetchMetalakes = createAsyncThunk('appMetalakes/fetchMetalakes', as
 
   const { metalakes } = res
 
+  metalakes.sort((a, b) => new Date(a.audit.createtime) - new Date(b.audit.createtime))
```

Comment:
```suggestion
  metalakes.sort((a, b) => new Date(b.audit.createTime) - new Date(a.audit.createTime))
```
I think we should use descending order by time, and the field name should be `createTime`.